### ISSUE TITLE
SMS Code

### ISSUE THREAD

**Issue by pfcpinto** [A-Georgiou/SimpleNikeAccountCreator#1]
Can't get SMS code from getsmscode, changed the pid to 628 also which is new nike and still inop.

**Comment by pfcpinto** [comment]
never mind works flawless if ran through the Chinese version.

**Comment by A-Georgiou** [comment]
Yeah the chinese version works great, I contacted the host of getsmscode about UK and US numbers as he was having issues all seems to work now

**Comment by pfcpinto** [comment]
Will proxy support be added?

**Comment by A-Georgiou** [comment]
Yeah I will post the new versions with that in the next few days with proxy support

**Comment by A-Georgiou** [comment]
If you want to add a proxy per launch short term you can swap out the line with "const browser" with "const browser = puppeteer.launch({ args: ['--proxy-server="socks5://INSERTPROXYHERE:8080'] });"
that will offer a temp solution

**Comment by gpicon6** [comment]
We are unable to connect to our servers right now. Please try again later.
{"code":"GENERIC","httpStatus":0}

is this an issue with the script? ive switched proxies 3 times, and cant get passed the create account window

**Comment by A-Georgiou** [comment]
have you managed to test if the proxies are active, https://proxy6.net/en/checker
Just tell me the results of your proxies on that if they are active I will pm you and look through your code

**Comment by A-Georgiou** [comment]
Like pfcpinto said I changed the pid over to the new getsmscode id which should definetely help.
although gpicon6 your issue seems to be with connecting to the server and getting a http response so I will take a look at that shortly

**Comment by Zero-01010** [comment]
Anyway to add my own list of emails and passwords for it to create accounts through ?

**Comment by alexforce123** [comment]
is possible to put an array of proxies? how can it automatically switch from first to end and then restart from the top? and uk works fine for me, cn cant access to created account on nik!!
someone can help
